pyproject.toml

```diff
@@ -39,7 +39,7 @@ streamlit = {version = "^1.23.1", optional = true}
 text-generation = { version = ">=0.6.0", optional = true }
 openpyxl = { version = "^3.0.7", optional = true }
 pymysql = { version = "^1.1.0", optional = true }
-psycopg2 = { version = "^2.9.7", optional = true }
+psycopg2-binary = { version = "^2.9.7", optional = true }
```

Comment:
See the details in https://github.com/Sinaptik-AI/pandas-ai/issues/1032 on why this is changed.